app/views/layouts/application.html.haml

```diff
@@ -10,3 +10,11 @@
   %body
     %h1 Terra UI
     = yield
+    :javascript
```

Comment:
Should this go into the head instead of the body? 

```
Add the entire tracking code snippet to each web page you want to track. Add the tracking code snippet just before the closing </head>tag on each page.
```
https://support.google.com/analytics/answer/1008080?hl=en#GA